README.md

```diff
@@ -0,0 +1,34 @@
+# A guide on how to use Try Enarx 🎯
+
+A little amount of efforts are required but we will walk you through all of it. For instance, trying enarx with your own WASM files is a matter of few clicks. Our goal is to understand how it happens step by step by *learning and doing*.
```

Comment:
We will walk you through how to try *Try Enarx*. For instance, trying Enarx with your own Wasm file is a matter of few clicks. Our goal is to understand how it happens step by step by *learning and doing*.